src/dialog-editor/components/modal/modalComponent.ts

```diff
@@ -58,7 +58,7 @@ class ModalController {
           };
           const titleLabel = this.modalData.type in titles &&
             titles[this.modalData.type];
-          this.modalTitle =  __(`Edit ${titleLabel} Field`);
+          this.modalTitle =  sprintf(__("Edit %s Field"), titleLabel);
```

Comment:
I'm guessing we have to add `sprintf()` somewhere for this to work, am I correct @himdel?

Reply:
Looks like both ui-classic and ui-service depend on sprintf and provide it on window.

So only the demo might fail on missing sprintf..

But `yarn add sprintf-js` plus:

```diff
diff --git a/src/vendor.ts b/src/vendor.ts
index 593d7cc..772ac71 100644
--- a/src/vendor.ts
+++ b/src/vendor.ts
@@ -17,3 +17,4 @@ import 'ui-select';
 import 'patternfly-bootstrap-treeview';
 import * as numeral from 'numeral';
 window['numeral'] = numeral;
+window['sprintf'] = require('sprintf-js').sprintf;
```

fixes that :)

Reply:
Done.